aten/src/ATen/function_wrapper.py

```diff
@@ -1060,11 +1082,21 @@ def find_formal(formal_name, formals):
         # Factory methods are not dispatched over `Type`.
         if not is_factory_method:
             if option['deprecated']:
+                # Deprecated functions are always non-extended,
```

Comment:
I'm not really sure what these have to do with each other, i.e. deprecated functions we definitely keep in type, but non-deprecated functions we think are fine to move to TypeExtendedInterface.  It's almost like we are using deprecated backwards...

Reply:
After in person discussion, action item is to remove the deprecated functions (methods on Type) in a later patch.